### ISSUE TITLE
global_step = 0?

### ISSUE THREAD

**Issue by 1041367296** [google-research/google-research#1717]
I am a newcomer, and after I managed to run bert-master, I found that when my run_classifier.py was run, its output would always be global_step = 0.
parameters:
--task_name=MRPC\
--do_train=true\
--do_eval=true\
--data_dir=..\GLUE\glue_data\MRPC\
--vocab_file=..\GLUE\BERT_BASE_DIR\uncased_L-12_H-768_A-12\vocab.txt\
--bert_config_file=..\GLUE\BERT_BASE_DIR\uncased_L-12_H-768_A-12\bert_config.json\
--init_checkpoint=..\GLUE\BERT_BASE_DIR\uncased_L-12_H-768_A-12\bert_model.ckpt\
--max_seq_length=128\
--train_batch_size=8\
--learning_rate=2e-5\
--num_train_epochs=3.0\
--output_dir=/tmp/mrpc_output/

I0816 18:35:17.464866 14584 run_classifier.py:924] ***** Eval results *****
INFO:tensorflow:  eval_accuracy = 0.64705884
I0816 18:35:17.464866 14584 run_classifier.py:926]   eval_accuracy = 0.64705884
INFO:tensorflow:  eval_loss = 0.63068485
I0816 18:35:17.464866 14584 run_classifier.py:926]   eval_loss = 0.63068485
INFO:tensorflow:  global_step = 0
I0816 18:35:17.465864 14584 run_classifier.py:926]   global_step = 0
INFO:tensorflow:  loss = 0.63068485
I0816 18:35:17.465864 14584 run_classifier.py:926]   loss = 0.63068485
enviroment:
tensorflow-gpu:2.5.0
python:3.7

**Comment by 1041367296** [comment]
***** Eval results *****
I0816 18:35:17.464866 14584 run_classifier.py:924] ***** Eval results *****
INFO:tensorflow:  eval_accuracy = 0.64705884
I0816 18:35:17.464866 14584 run_classifier.py:926]   eval_accuracy = 0.64705884
INFO:tensorflow:  eval_loss = 0.63068485
I0816 18:35:17.464866 14584 run_classifier.py:926]   eval_loss = 0.63068485
INFO:tensorflow:  global_step = 0
I0816 18:35:17.465864 14584 run_classifier.py:926]   global_step = 0
INFO:tensorflow:  loss = 0.63068485
I0816 18:35:17.465864 14584 run_classifier.py:926]   loss = 0.63068485
